builders/build/substrate-api/sidecar.md

```diff
@@ -351,3 +351,7 @@ And then `Transaction Weight` is mapped to the following field of the block JSON
 ```
 extrinsics.{extrinsic number}.events.{event number}.data.0.weight
 ```
+
+## Sample Code for Monitoring Native Token Transfers { #sample-code-for-monitoring-native-token-transfers }
```

Comment:
Would it make sense to say that this is including transfers using both the Substrate and Ethereum APIs?